src/index.js

```diff
@@ -179,6 +179,20 @@ export default class ScrollBehavior {
     scrollTop(element, y);
   }
 
+  _defaultScrollTarget(hash) {
```

Comment:
call this `_getDefaultScrollTarget` and make it take `location` rather than just `hash`